### PR TITLE
43 - client and server modules are separated

### PR DESCRIPTION
**Pull request by DConnorFong**
client and server modules are separated and communicate with a proxy (tested)

run 'npm start' on both modules and a communication success message should display on page

### CODE REVIEW

**Review by cowmanjoe — DISMISSED** [review]
Good job on this! Got it working on my machine. One thing is I think the old package-lock.json is still present in the top level directory even though there is no package.json. I think this is unnecessary and was causing me some errors. Upon deletion it worked like a charm. I'm only requesting changes for that one file, everything else looks good to me.

File: client/src/css/App.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.App {
```
Comment (frostyshadows) [inline client/src/css/App.css:1]: have you considered using SCSS instead? It has all the functionality of CSS with a few other cool features, for example being able to use variables.

Comment (DConnorFong) [reply client/src/css/App.css:1]: Yeah, I would like to use SCSS - I'll add a ZenHub ticket for it. It didn't seem like it was necessary for this case, so I omitted it from these changes.

Comment (frostyshadows) [reply client/src/css/App.css:1]: Sounds good, we can add it when we start needing to style the frontend.

Comment (DConnorFong) [reply client/src/css/App.css:1]: Ticket added to backlog!

**Review by cormacjmollitor — APPROVED** [review]

**Pull request merged**